### PR TITLE
Submitting HappiTails as a folder without .git/submodule

### PR DESCRIPTION
**Pull request by fashed**
Hope this works!

### CODE REVIEW

**Comment by trivett** [comment]
@fashed I'll hold off on this until you push up your work from this afternoon.

**Comment by fashed** [comment]
Pushed up work from today. Thanks Vincent!

File: jennifer_choi/HappiTails/lib/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@HappiTails = Shelter.new("HappiTails")
```
Comment (trivett) [inline jennifer_choi/HappiTails/lib/seeds.rb]: This is one way to make the shelter kind of global, but might run into namespace issues. If this is to be a one-shelter app you can make globals like: `$HAPPITAILS`

anther way would be to put the seed stuff into a method that returns the shelter, then in main, set a variable to calling that method.

File: jennifer_choi/HappiTails/main.rb [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	puts "#{client.name} wants to adopt."
+
+	puts "Which pet is being adopted?"
+	@HappiTails.show_animals
+	p = gets.chomp.to_i
+	pet = @HappiTails.animals[p - 1]
+
+
+ 	#delete from Shelter
+ 	@HappiTails.animals.delete_at(p - 1)
```
Comment (trivett) [inline jennifer_choi/HappiTails/main.rb]: nice method chaining!

**Comment by trivett** [comment]
Great work on the midterm @fashed. You are doing really well in class. If there is anything you want to hear explained or anything, you can always hit me up on slack. See you tomorrow!

**Pull request merged**